build_backend.py

```
@@ -25,6 +29,655 @@
+        setup_args += [
+            "-Dbuild_examples=false",
+            "-Dnixl_ep_only=true",
+            f"-Dnixl_wheel_lib_dir={wheel_lib_dir}",
+        ]
+        print(
+            f"[BUILD_NVEP] nixl_ep_only=true; linking against wheel libnixl.so "
+            f"at {wheel_lib_dir}. Set BUILD_NIXL_EP_HERMETIC=1 to opt out."
+        )
+
+    if not build.exists():
+        subprocess.run(setup_args, check=True)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)